index.html

```diff
@@ -229,10 +229,56 @@ <h2 class="project-title">Uber Navigation</h2>
               </ul>
             </div>
             <div class="action">
-              <button type="button" class="action-button">See Project</button>
+              <button onclick="a(4)" id="btn-4" type="button" class="action-button">See Project</button>
             </div>
           </div>
         </div>
+        <!-- Card-aux -->
+      <div class="container-card-aux">
+        <div class="card-aux">
+          <div class="header-aux">
+            <div class="title-aux">
+              <h2 id="title-a">Aux</h2>
+              <button id="aux-close" class="close-aux"><img src="./img/btn-close-black.svg"></button>
+            </div>
+            <div class="frame-2">
+              <div class="client">
+                <p class="text-p" id="tc-a">Aux</p>
+              </div>
+              <div class="counter"></div>
```

Comment:
- Great job for having a good design 👍👍 I think it would be better to have an exact blur, font size, and size for your list as the Figma design to have better design 👍👍



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/217462008-0d7bb90f-10d0-4bec-8337-0ce0579dc372.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/217462026-1f1e858c-7479-4274-968b-d90d1abbad8d.png" width="600">|


😎️
🤝️

Reply:
> Correction

I am sorry for not following the Figma template.

I invite you to review my work again. 😎🤘